documentation/modules/upgrading/proc-upgrade-cruise-control-api-version.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cruise-control-api-version-{context}']
+= Upgrading Cruise Control to support {ApiVersion}
```

Comment:
Were we going to use _Manually_ in these titles to distinguish from the new tool approach?

Reply:
I don't think we need to. Let's see how the content looks once it's merged in.